app/services/workflow_delete_service.rb

```diff
@@ -9,6 +9,7 @@ def destroy
     begin
       retries ||= 0
       Workflow.find(workflow_id).destroy!
+      EventService.new(nil).workflow_deleted(workflow_id)
```

Comment:
I know you just moved this, so I'm not going to hold up the PR for it, but it feels weird that we're passing in `nil` here. I think we should just use `initialize(request = nil)` in the definition so that we would do `EventService.new.workflow_deleted` here. That can be a separate small refactor change, though.